.github/workflows/tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install flake8==6.0
+          pip install flake8
```

Comment:
Previously had to set the version due to a bug, but that has been corrected with the current flake8.